README.md

```diff
@@ -10,9 +10,14 @@ The CockroachDB Kubernetes Operator deploys CockroachDB on a Kubernetes cluster.
 ## Limitations
 
 - This project is in beta and is not production-ready.
-- The Operator currently runs on GKE.
+- The Operator currently runs on GKE. VMware Tanzu, EKS, and AKS have not been tested.
 - Currently the Kubernetes CA is required to deploy a secure cluster.
 - Expansion of persistent volumes is not yet functional.
+- The Operator has not been tested with [Istio](https://istio.io/).  
```

Comment:
Do we need to mention this? Really is not pertinent

Reply:
I'll defer to @johnrk as it was on his list. I moved it to the bottom of this list.

Reply:
@taroface , yes let's list it. Customers consistently ask about it.